Makefile

```diff
@@ -45,7 +45,7 @@ kube-delete-database:
 
 kube-delete-all: kube-delete kube-delete-database
 
-kube-create:
+kube-create: update-manifests
```

Comment:
Because not all usages of `kube-create` want the image name to be munged. I'll defer to @sgoings for consistency across projects, but I know that's the reason why I didn't originally make kube-create dependency on update-manifests.